qiskit/visualization/state_visualization.py

```diff
@@ -330,19 +328,140 @@ def plot_state_city(rho, title="", figsize=None, color=None,
+                      oper_name=None, basis=None):
+    """Plot the cityscape of quantum channel.
+
+    Plot two 3d bar graphs (two dimensional) of the real and imaginary
+    part of the channel.
+
+    Args:
+        channel (ndarray): Numpy array representing quantum channel.
+        title (str): a string that represents the plot title
+        figsize (tuple): Figure size in inches.
+        color (list): A list of len=2 giving colors for real and
+            imaginary components of matrix elements.
+        alpha (float): Transparency value for bars
+        ax_real (matplotlib.axes.Axes): An optional Axes object to be used for
+            the visualization output. If none is specified a new matplotlib
+            Figure will be created and used. If this is specified without an
+            ax_imag only the real component plot will be generated.
+            Additionally, if specified there will be no returned Figure since
+            it is redundant.
+        ax_imag (matplotlib.axes.Axes): An optional Axes object to be used for
+            the visualization output. If none is specified a new matplotlib
+            Figure will be created and used. If this is specified without an
+            ax_imag only the real component plot will be generated.
+            Additionally, if specified there will be no returned Figure since
+            it is redundant.
+        oper_name (str): Representation of channel to show in z-axis label
+        basis (tuple[str]): A list of string to represent each basis operator.
+
+    Returns:
+         matplotlib.Figure:
+            The matplotlib.Figure of the visualization if the
+            ``ax_real`` and ``ax_imag`` kwargs are not set
+
+    Example:
+        .. jupyter-execute::
+
+           from qiskit import QuantumCircuit
+           from qiskit import quantum_info as qi
+           from qiskit.visualization import plot_process_city
+           %matplotlib inline
+
+           qc = QuantumCircuit(2)
+           qc.cx(0, 1)
+
+           plot_process_city(qi.Chi(qc).data, color=['midnightblue', 'midnightblue'],
+                title="New Process City")
+
+    """
+    basis = basis or ['i', 'x', 'y', 'z']
+    num = int(math.log(channel.shape[0], len(basis)))
+
+    # get the labels
+    row_names = list(map(''.join, itertools.product(basis, repeat=num)))
+    column_names = list(map(''.join, itertools.product(basis, repeat=num)))
+
+    fig = _plot_matrix_city(mat=channel, row_names=row_names, column_names=column_names,
+                            oper_name=oper_name or '$\\chi$', title=title, figsize=figsize,
+                            color=color, alpha=alpha, ax_real=ax_real, ax_imag=ax_imag)
+
+    if ax_real is None and ax_imag is None:
+        if get_backend() in ['module://ipykernel.pylab.backend_inline',
+                             'nbAgg']:
+            plt.close(fig)
+        return fig
+
+
+def _plot_matrix_city(mat, row_names, column_names, oper_name,
```

Comment:
This could be a useful function to make available to users with signature:

```python
def plot_matrix_city(mat, row_labels=None, col_labels=None, oper_name='matrix',  ...)
    """Plot the cityscape of arbitrary matrix.
    Plot two 3d bar graphs (two dimensional) of the real and imaginary
    part of a complex matrix.
    Args:
        mat (ndarray): Numpy array of matrix to visualize.
        row_labels (list(str)): Labels for row. If `None` use integer labels.
        column_names (list(str)): Labels for column. If `None` use integer labels.
        row_labels (list(str)): Labels for row. If `None` use integer labels.
        column_names (list(str)): Labels for column. If `None` use integer labels.
        <rest the same>  
```

Then `plot_state_city` and the new function will just be this with appropriate axis labels